vendor-extra/ViewsBundle/tests/Views/StringViewConverterTest.php

```diff
@@ -0,0 +1,79 @@
+<?php
+
+declare(strict_types=1);
+
+namespace tests\Libero\ViewsBundle\Views;
+
+use Libero\ViewsBundle\Views\CallbackViewConverter;
+use Libero\ViewsBundle\Views\StringView;
+use Libero\ViewsBundle\Views\StringViewConverter;
+use Libero\ViewsBundle\Views\TemplateViewBuildingViewConverter;
+use Libero\ViewsBundle\Views\View;
+use Libero\ViewsBundle\Views\ViewConverter;
+use PHPUnit\Framework\TestCase;
+use Symfony\Component\EventDispatcher\EventDispatcherInterface;
+use tests\Libero\LiberoPageBundle\ViewConvertingTestCase;
+use tests\Libero\LiberoPageBundle\XmlTestCase;
+
+final class StringViewConverterTest extends TestCase
+{
+    use ViewConvertingTestCase;
+    use XmlTestCase;
+
+    /**
+     * @test
+     */
+    public function it_is_a_view_converter() : void
+    {
+        $converter = new StringViewConverter($this->createFailingConverter());
+
+        $this->assertInstanceOf(ViewConverter::class, $converter);
+    }
+
+    /**
+     * @test
+     * @dataProvider visibleNodeProvider
+     */
+    public function it_returns_a_string_view_for_visible_nodes(string $node, string $expected) : void
+    {
+        $converter = new StringViewConverter($this->createFailingConverter());
```

Comment:
from the test it isn't clear whether the `fallback` inner converter is not called or whether its failure is caught; I'd substitute it with a mock that specified `$this->never()` if that's the intention

Reply:
Looks like the relevant test was broken, fixed in f1b0bdede0c2cf113e04346c447128b5ebfb4b64.